packages/next/taskfile.js

```diff
@@ -77,6 +77,7 @@ const externals = {
   'terser-webpack-plugin':
     'next/dist/build/webpack/plugins/terser-webpack-plugin/src',
 
+  punycode: 'punycode/',
```

Comment:
how come this is the only one with a trailing slash?

Reply:
trailing slash enforces the use of external [punycode](https://github.com/mathiasbynens/punycode.js) library, instead of the node-embedded one